clients/orderbook/src/worker.rs

```diff
@@ -155,6 +157,8 @@ pub(crate) struct ObWorker<B: Block, BE, C, SO, N, R> {
 	last_processed_block_in_offchain_state: BlockNumber,
 	// Version of current state
 	state_version: Arc<RwLock<u16>>,
+	// is runtime upgraded and client can process
```

Comment:
```suggestion
	/// is runtime upgraded and client can process
```
I'd suggest use doc syntax (e.g. `///`) here on pub scope as on other pub members...